metadata-models/src/main/pegasus/com/linkedin/dataJob/azkaban/AzkabanJobInfo.pdl

```diff
@@ -0,0 +1,19 @@
+namespace com.linkedin.dataJob.azkaban
+
+import com.linkedin.common.AzkabanFlowUrn
+
+/**
+ * Metadata associated with a Azkaban job
+ */
+record AzkabanJobInfo {
+
+  /**
+   * Reference to the parent Azkaban flow
+   */
+  flow: AzkabanFlowUrn
```

Comment:
Don't think we need this bi-directional relationship between job & flow. What's the rationale?